tests/integration/test_integration_artifactory_path.py

```diff
@@ -173,6 +175,81 @@ def test_deploy_file(path):
     p.unlink()
 
 
+def test_deploy_file_by_checksum(path):
+    p = path("/integration-artifactory-path-repo/foo")
+    p1 = path("/integration-artifactory-path-repo/bar")
+
+    if p.exists():
+        p.unlink()
+    if p1.exists():
+        p1.unlink()
+
+    tf = tempfile.NamedTemporaryFile()
```

Comment:
The upload code should be in a pytest.fixture

Reply:
Fixed